### PR TITLE
send load analytics event on initialization

### PR DESCRIPTION
**Pull request by domenicrosati**
We want to track customer analytics over who loads the page, when, where, and why.

We could go through all of our components and manually add this event OR we could through the load event when the connection is initialized once and it will work for all components that use this version of manifold-init.

# How it works?

Do a track event over component load on initialization 

# Demo:

Screen Shot 2020-05-08 at 12 38 22 PM
Screen Shot 2020-05-08 at 12 38 06 PM
Screen Shot 2020-05-08 at 12 35 28 PM
Screen Shot 2020-05-08 at 12 35 23 PM

### CODE REVIEW

File: src/v0/__tests__/index.spec.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        element: document.createElement('manifold-product'),
+        getAuthToken: () => '',
+        getOwnerId: () => '',
+        clearAuthToken: () => {},
+        clientId: '123',
+      });
+      expect(fetchMock.called(endpoint.local)).toBe(true);
+      const res = fetchMock.calls()[0][1];
+      expect(res && res.body && JSON.parse(res.body.toString())).toEqual({
+        source: 'MANIFOLD-PRODUCT',
```
Comment (dangodev) [inline src/v0/__tests__/index.spec.ts:24]: Why is this the source?

Comment (dangodev) [reply src/v0/__tests__/index.spec.ts:24]: Oh oh it’s just a test nvm

Comment (domenicrosati) [reply src/v0/__tests__/index.spec.ts:24]: ya well its also a good question on the source v. component.....

Source being the higher level namespace and the component being the actual HTMLComponent. Philisophically maybe that isn't really useful without composable components.

I can see us having multiple composable components in the future such as MANIFOLD-PLAN which would be the component and the source could be a parent such as MANIFOLD-PRODUCT or something... for now components source and component are the same.

Comment (dangodev) [reply src/v0/__tests__/index.spec.ts:24]: Yeah that makes sense. We can always adjust analytics and get more granular as-needed for sure (obviously you want to do that as soon as you realize something but also you probably won’t get it 100% correct first pass either).

**Pull request merged**